### PR TITLE
ath79: add support for Ubiquiti PowerBridge M

### PR DESCRIPTION
**Pull request by txt-file**
sadly wifi-LED is not working, help very much welcome
proper commit message missing
wireless tested and is working
ethernet tested and is working
https://openwrt.org/toh/ubiquiti/airmaxm?s[]=powerbridge

### CODE REVIEW

File: target/linux/ath79/image/generic-ubnt.mk [hunk 1]
```diff
@@ -269,6 +269,15 @@ define Device/ubnt_powerbeam-5ac-gen2
 endef
 TARGET_DEVICES += ubnt_powerbeam-5ac-gen2
 
+define Device/ubnt_powerbridge-m
+  $(Device/ubnt-xm)
+  SOC := ar7241
+  DEVICE_MODEL := PowerBridge-M
+  DEVICE_PACKAGES += rssileds
+  SUPPORTED_DEVICES += powerbridge-m ubnt,powerbridge-m
```
Comment (adrianschmutzler) [inline target/linux/ath79/image/generic-ubnt.mk:277]: This line can be dropped. Was there an image available for ar71xx?

File: target/linux/ath79/image/generic-ubnt.mk [hunk 2]
```diff
@@ -269,6 +269,15 @@ define Device/ubnt_powerbeam-5ac-gen2
 endef
 TARGET_DEVICES += ubnt_powerbeam-5ac-gen2
 
+define Device/ubnt_powerbridge-m
+  $(Device/ubnt-xm)
+  SOC := ar7241
+  DEVICE_MODEL := PowerBridge-M
```
Comment (adrianschmutzler) [inline target/linux/ath79/image/generic-ubnt.mk:275]: Use space instead of hyphen.

**Comment by adrianschmutzler** [comment]
> sadly wifi-LED is not working, help very much welcome

Yes, because you did not set it up.
network setup is missing as well.

> proper commit message missing

Yes, please add it.

File: target/linux/ath79/image/generic-ubnt.mk (hunk 1 above)
Comment (txt-file) [reply target/linux/ath79/image/generic-ubnt.mk:277]: ar71xx_bullet-m  is working on the device. Shall I add that?

Comment (adrianschmutzler) [reply target/linux/ath79/image/generic-ubnt.mk:277]: `SUPPORTED_DEVICES += bullet-m`

**Comment by txt-file** [comment]
done from my side :-)
* LED working now
* proper commit message added

**Comment by adrianschmutzler** [comment]
Thanks, merged into my staging tree.

**Pull request closed without merging**